### ISSUE TITLE
Remove pre-existing xcargs `-parallel-testing-enabled` in multi_scan

### ISSUE THREAD

**Issue by andreaantonioni** [lyndsey-ferguson/fastlane-plugin-test_center#189]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description
When I use `multi_scan` with the param `parallel_testrun_count` > 0, I see in the logs that in `xcargs` a double `parallel_testrun_count` param. 
I think is due to this: https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/578de6358c08b763b98d335b25cc48179bbdbba6/lib/fastlane/plugin/test_center/helper/multi_scan_manager/retrying_scan_helper.rb#L94

Is it right that there's a double param?

**Comment by lyndsey-ferguson** [comment]
@andreaantonioni thanks for the question. I'm not certain I understand the question.

Are you stating that you see `-parallel-testing-enabled NO` *twice* in the logs for the call to `xcodebuild test`? I think it should not harm it. I am curious why that is happening. Can you include the logs and a copy (anonymized if you like) of your `Fastfile` (specifically the lane that calls `multi_scan`).

**Comment by andreaantonioni** [comment]
Okay, but I think`-parallel-testing-enabled NO` should be `-parallel-testing-enabled=NO`, isn't it?

Then when I run this:
```
multi_scan(
        workspace: "Example.xcworkspace",
        try_count: 1,
        fail_build: false,
        scheme: "UITests",
        device: "iPhone X (iOS 13.3)",
        testrun_completed_block: testRerunBlock,
        xcargs: "-UseModernBuildSystem=NO",
        result_bundle: true,
        parallel_testrun_count: 4
    )
```

from the logs I see this:
`[worker 3] [0m| xcargs | -UseModernBuildSystem=NO -parallel-testing-enabled=YES  -parallel-testing-enabled NO `

**Comment by lyndsey-ferguson** [comment]
Hmm. The docs suggest without the equals:
```
    -parallel-testing-enabled YES|NO                         overrides the per-target setting in the scheme
```
I don't see it being turned on by `fastlane`. Have you turned it on from your Project?

**Comment by andreaantonioni** [comment]
Ops, I just found my mistake. I was adding manually `-parallel-testing-enabled=YES` in `xcargs` in `multi_scan`.
Sorry 🙏🏻

**Comment by lyndsey-ferguson** [comment]
@andreaantonioni no worries, I will update the code to remove the regex `-parallel-testing-enabled(=|\s)(YES|NO)` before applying `-parallel-testing-enabled NO`

**Comment by lyndsey-ferguson** [comment]
@andreaantonioni would you like to test this for me?

If so, can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again (with the `--verbose` flag)?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-189-override-parallelism-xcargs"
```

If there are still problems, please let me know and attach the console output as a text file to this issue (makes it easier for me to review).  If it works, please let me know.

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.10.0](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.10.0)
